src/au3cloud/internal/au3cloudactionscontroller.cpp

```
@@ -39,3 +43,23 @@ void Au3CloudActionsController::openSignInDialog(const muse::actions::ActionQuer
         platformInteractive()->openUrl(TOUR_PAGE_URL);
     }
 }
+
+void Au3CloudActionsController::openCloudProjectPage(const muse::actions::ActionQuery& query)
+{
```

Cppcheck (2.20.0)

[style] The function 'availableTrackFormats' is never used.

(unusedFunction)